common/test/acceptance/pages/lms/teams.py

```diff
@@ -18,4 +19,9 @@ def is_browser_on_page(self):
 
     def get_body_text(self):
         """ Returns the current dummy text. This will be changed once there is more content on the page. """
-        return self.q(css='.page-content-main').text[0]
+        main_page_content_css = '.page-content-main'
+        EmptyPromise(
+            lambda: len(self.q(css=main_page_content_css).text) == 1,
```

Comment:
I am checking the length of the array to make sure there is one element (though I admit I have not tried running this test locally).